File: movieGenre.py
class Movie:
    uuid = ""
    title = ""
    year = ""
    genres = None
    related = []

    def addRelatedMovie(self, m):
        if isinstance(m, Movie):
            related.append(m)
            return true
        return false

    def setGenre(self, g):
        if isinstance(g, Genre):
            genres = g
            g.addMovie(self)
            return true
        return false

class Genre:
    name = ""
    movies = []

    def addMovie(self, m):
        if isinstance(m, Movie):
            movies.append(m)
            return true
        return false

File: movieGenre.js
class Movie {
    constructor(){
        this.uuid = "";
        this.title = "";
        this.year = "";
        this.genres = null;
        this.related = [];   
    }
   addRelatedMovie(m){
        if(m instanceof Movie){
            related.push(m);
            return true;
        }
        return false;
    }
    setGenre(g){
        if(g instanceof Genre){
            genres = g;
            g.addMovie(this);
            return true;
        }
        return false;
    }

}

class Genre{
    constructor(){
        this.name = "";
        this.movies = [];
    }
    addMovie(m){
        if(m instanceof Movie){
            movies.push(m);
            return true;
        }
        return false;
    }
}


